### ISSUE TITLE
Japanread bug

### ISSUE THREAD

**Issue by Batmanji** [tachiyomiorg/tachiyomi-extensions#9813]
### Source information

Japanread

### Steps to reproduce

1. Click the manga
2. Click the chapter
3. Message pop "aucune page trouvé"

### Expected behavior

By read the manga on this source because in I have a manga is not other source

### Actual behavior

I don't understand

### Tachiyomi version

0.12.3

### Android version

Android 10

### Other details

Hello, japanread, french source is down, what can you fix the source please
Thanks
(Sorry my English, im french)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Adam935** [comment]
it's look like it's fix so u can close it 

il me semble que c'est bon don't tu peux fermer l'issue

**Comment by tdf27** [comment]
Reading blocked by Cloudflare

**Comment by arkon** [comment]
In an effort to have a more manageable issue backlog, we're closing older requests that weren't addressed since there's a low chance of it being addressed if it hasn't already. If your request is still relevant, please [open a new request](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
